includes/wc-template-functions.php

```diff
@@ -704,7 +731,9 @@ function wc_product_class( $class = '', $product_id = null ) {
  */
 function wc_query_string_form_fields( $values = null, $exclude = array(), $current_key = '', $return = false ) {
 	if ( is_null( $values ) ) {
-		$values = $_GET; // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+		// phpcs:disable WordPress.Security.NonceVerification.Recommended
+		$values = $_GET;
```

Comment:
It should work with just one line using `phpcs:ignore WordPress.Security.NonceVerification.Recommended`

Reply:
I could swear that I did it that way because it wasn't accepting the one-liner version. But now it is accepting it. 🤷🏻‍♂️